### ISSUE TITLE
OBS crashes on specific events in other applications

### ISSUE THREAD

**Issue by akamensky** [obsproject/obs-studio#3487]
## Platform

Operating system and version: macOS 1.15.6
OBS Studio version: 24.0.6 (cannot upgrade to 25.x because of insanely high CPU usage see #2841 for that)

## Expected Behavior
OBS works fine and does not crash.

## Current Behavior
OBS reliably and reproducibly crashes on some events in other applications. First I thought it was random. But then I found stable reproducible pattern to that with 1Password (v7.x)

## Steps to Reproduce
1. Have OBS running (recording or streaming or both) with an active display capture in ANY of the scenes
2. Have 1Password locked
3. On any screen (either captured or not -- does not matter) activate 1Password and type-in the password to unlock it
4. Hit enter (to unlock after typing-in the password)
5. Check OBS (it is gone, no stacktrace, no any notification, it is just gone the moment I hit enter to unlock 1Password)

**Comment by WizardCM** [comment]
The password crash is fixed in 25.0.8, it has to do with secure inputs. Please try [the latest RC](https://github.com/obsproject/obs-studio/releases/tag/26.0.0-rc3) to see if performance is better. Otherwise, keep an eye on that other ticket.

**Comment by akamensky** [comment]
Ah good to know my search did not find this previous ticket.

**Comment by akamensky** [comment]
@WizardCM can this fix be backported to 24.x? I tried 26-RC3 and it still high CPU there (I added my comment to the relevant ticket).

My concern is that 26.x will possibly still have performance issue on macos when released, in which case I am (and others) still stuck on 24.x. But we would need to make sure that things at least do not crash unexpectedly mid-stream or during the long recording session.
